day3/slice_time_image.py

```diff
@@ -82,12 +114,32 @@ def main():
     fname = sys.argv[1]
     # Assume the TR
     TR = 2.0
+    
     # Assume the slices were acquired even slices first, inferior to
     # superior, then odd slices, inferior to superior, where the most inferior
     # slice is index 0 and 0 is an even number.  What are the slice acquisition
     # times in seconds, where the first value is the acquisition time of slice
     # 0, the second is acquisition time of slice 1, etc?
-    slice_times = ?
+    
+    # have to load in image to know the number of slices there are
+    img = nib.load(fname)
+    num_slices = img.shape[2]
```

Comment:
Good - I was being sloppy and assuming you would hard-code the slice times, but this is a lot better.